syfertext/pointers/token_pointer.py

```diff
@@ -16,8 +16,6 @@ def __init__(
         owner: BaseWorker = None,
         id: Union[str, int] = None,
         garbage_collect_data: bool = True,
```

Comment:
please remove `garbage_collect_data` too. It is `True` by default in the parent class

Reply:
Done.